src/Nethermind/Nethermind.Specs/Forks/17_Cancun.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Nethermind.Specs.Forks
 {
-    public class Cancun : Shanghai
+    public class Cancun : ShardingFork
```

Comment:
Isn't Cancun same as SHardingFork?

Reply:
I will remove it in favor to Cancun, but remove transient opcodes from Cancun just in 4844 branch, because I don't think they should be included for now.